Resources/Public/JavaScript/Main.js

```diff
@@ -0,0 +1,36 @@
+document.addEventListener("DOMContentLoaded", function () {
+    var t = document.querySelectorAll(".tx-tea table"),
+        i = t.length;
+    while (--i >= 0) makeSortable(t[i]);
+});
+
+function makeSortable(table) {
```

Comment:
Should we namespace the functions to avoid name collisions, e.g., like this?
https://github.com/oliverklee/ext-seminars/blob/master/Resources/Public/JavaScript/FrontEnd/FrontEnd.js#L22